src/utils/url.ts

```diff
@@ -40,20 +40,6 @@ export const openRoomUrl = (url: string) => {
 export const enterVenue = (venueId: string) => openUrl(venueInsideUrl(venueId));
 
 export const openUrl = (url: string) => {
-  if (!isValidUrl(url)) {
-    Bugsnag.notify(
-      // new Error(`Invalid URL ${url} on page ${window.location.href}; ignoring`),
-      new Error(
-        `Invalid URL ${url} on page ${window.location.href}; allowing for now (workaround)`
-      ),
-      (event) => {
-        event.addMetadata("utils/url::openUrl", { url });
-      }
-    );
-    // @debt keep the checking in place so we can debug further, but don't block attempts to open
-    // return;
-  }
-
```

Comment:
I believe we want to keep this (particularly the Bugsnag error reporting bit) so that we know if/when data is invalid (eg. all of the existing data in the database that may not match the correct URL format)

But we likely want to get rid of the `@debt` part and restore the `return` that 'bails out early' when it isn't a valid URL.

Reply:
restored the Bugsnag error reporting and fixed return statement